transifex-migrate-resource.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python
+'''
+Run the script providing the slug of the project and the slug of the old and new resource.
+The new resource should already have been created.
+ 
+Example:
+Old resource slug: 'old'
+New resource slug: 'new'
+ 
+python transifex-migrate-resource.py project old new
+ 
+After running the command you will be asked for your Transifex username and password.
+'''
+# W.J. 2015 (license: MIT)
+# Based on https://gist.github.com/mapapage/bb07703afaa4ea590109 by Marilena Papageorgiou 
+
+from hashlib import md5
+import requests
+from requests.exceptions import RequestException
+import json
+import getpass
+import sys
+
+
+SERVER = "https://www.transifex.com"
+
+
+def get_source_entity_hash(context, key):
```

Comment:
```suggestion
def get_source_entity_hash(context, key) -> str:
```

Reply:
Let's do these kind of improvements separately.